Python/Product/Debugger/Debugger/DebugEngine/Remote/PythonRemoteDebugProgram.cs

```diff
@@ -85,7 +86,9 @@ internal class PythonRemoteDebugProgram : IDebugProgram2 {
         }
 
         public int GetEngineInfo(out string pbstrEngine, out Guid pguidEngine) {
-            pguidEngine = AD7Engine.DebugEngineGuid;
+            pguidEngine = ExperimentalOptions.GetUseVsCodeDebugger() ? 
```

Comment:
The property is preferred over the getter function. These options require restarting VS, so they aren't going to change after launch.